tests/e2e/e2e_test.go

```diff
@@ -14,7 +14,45 @@ import (
 	"github.com/osmosis-labs/osmosis/v7/tests/e2e/util"
 )
 
-func (s *IntegrationTestSuite) TestQueryBalances() {
+func (s *IntegrationTestSuite) TestAIBCTokenTransfer() {
```

Comment:
The tests in e2e test are run in alphabetical order. Since we token transfer first and then query the balance, I prefaced them with TestA and TestB respectively. Please let me know if there is another way we should go about this.

Reply:
I just realized that the query balances fails when using distroless. Should I change the query function to instead use dockertest or should we just role with plain ubuntu for now? @p0mvn

Reply:
uhhh, can we find some other abbreviations here. `AIBC` looks way to close to `ABCI` :sweat_smile:

Reply:
Happy for that to be relegated to a separate github issue tho